lib/streamlit/elements/time_widgets.py

```diff
@@ -214,17 +228,17 @@ class TimeWidgetsMixin:
     def time_input(
```

Comment:
I tried, but didn't really find a way, can look at it in some spare time, however I'd like to put a dot on this, since I'm moving to s4t project

Reply:
> however I'd like to put a dot on this, since I'm moving to s4t project

We probably shouldn't rush this project since it touches many important parts of the codebase :( But it totally understandable that you want to move focus to s4t projects. I can offer to finish this project later this week or next, with you switching to the reviewer role for this PR.